stock_picking_operation_quick_change/wizards/stock_picking_wizard.py

```diff
@@ -0,0 +1,85 @@
+# -*- coding: utf-8 -*-
+# © 2017 Sergio Teruel <sergio.teruel@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openerp import _, api, fields, models
+from openerp.exceptions import UserError
+
+
+class StockPickingOperationWizard(models.TransientModel):
+    _name = 'stock.picking.operation.wizard'
+
+    def _prepare_deafault_values(self, picking):
```

Comment:
IMO, even if function names should be at developer choice, avoid spelling faults ('default')